tests/publiccloud/prepare_tools.pm

```diff
@@ -133,6 +132,10 @@ EOT
     # Remove persistent net rules, necessary to boot the x86_64 image in the aarch64 test runs
     assert_script_run('rm /etc/udev/rules.d/70-persistent-net.rules');
 
+    # Add marker file for PC tools image
+    my $PUBLISH_HDD_1 = get_var("PUBLISH_HDD_1", "");
```

Comment:
I would go for `get_required_var` here . if `prepare_tools` running in job which does not have `PUBLISH_HDD_1` var something going wrong

Reply:
On test runs I don't set this variable, and a failure of this kind at the end of the test run is more annoying than helpful. I'd say this check is not really useful, but if you want to have it, then it should be at the beginning of the test module, not at the end.

Reply:
yes please move this to the beginning of the module

Reply:
done.